examples/fewshot_protonet/main.py

```diff
@@ -0,0 +1,126 @@
+"""
+This example is about training prototypical networks to solve N-Way-K-Shot problems.
```

Comment:
Similar comments about English, try to use simpler direct English, here and at other places (check all): This example trains ...

Reply:
Changed

Reply:
What has changed? Read my comment and check, please.

Reply:
Changed to _This demo trains a prototypical network to solve N-Way-K-Shot problems._